io.gitlab.cyberphantom52.sudoku_solver.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "io.gitlab.cyberphantom52.sudoku_solver",
```

Comment:
Please use `id` instead of `app-id`.

Reply:
On it

Reply:
https://github.com/cyberphantom52/flathub/commit/c2dbed29b9bd4e55a7fe0f9f55214b6a1b9f3690 taken care of